CHANGES.rst

```diff
@@ -81,6 +89,26 @@ API changes
 
 - ``astropy.nddata``
 
+  - ``NDDataBase`` only defines an abstract ``uncertainty`` getter instead of
+    an actual implementation. The original was moved to ``NDData``.
+
+  - ``NDDataBase`` doesn't implement an ``uncertainty`` setter. It was moved to
+    ``NDData``. The setter was also modified:
+
+    - it wraps an ``uncertainty`` inside an ``UnknownUncertainty``
```

Comment:
Make each of these sub-bullets main level bullets. So:

`NDData` wraps the `uncertainty` inside an `UnknownUncertainty`...

and add the issue number.